whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/vcard/VCardType.java

```diff
@@ -3,7 +3,9 @@
 public enum VCardType {
 
     TEXT("text"),
-    URI("uri");
+    URI("uri"),
+    WORK("work");
+
```

Comment:
We should not add work or home in my opinion, we do not know for sure, better to keep it as it is and RFC says  may be values not must or only vales .